documentation/commands.md

```diff
@@ -1,400 +1,13 @@
 # Command Reference
 
-Although the tutorial gives a large amount of information, if you find
-remembering the commands too complicated, here's a reference. Remember
-that you can type out multiple commands in one go; for example:
-`BAW231 fh 090 d 30 sp 180` will work as well as all three commands run
-separately. Some commands have "aliases" that are shorter to type. An
-example of that would be the `takeoff` command which has an alias `to`.
+Here you can find a list of all commands available in the game, what they do, and how to use them. There are two different sets of commands: _aircraft commands_ and _scope commands_. To toggle between entering aircraft commands and scope commands, use the `tab` key.
```

Comment:
I really try to avoid the word `game` and instead use or sim (preferred).  That's what it is after all, a simulator. Not a game.

Reply:
^ Man after my own heart....

Fixed.